btree/btree.c

```diff
@@ -5360,29 +5366,38 @@ static int64_t btree_del_kv_tick(struct m0_sm_op *smop)
 			}
 			return btree_del_resolve_underflow(bop);
 		}
-		lock_op_unlock(tree);
-		return m0_sm_op_sub(&bop->bo_op, P_CLEANUP, P_FINI);
+		M0_ASSERT(0);
 	}
+	case P_CAPTURE:
+		/**
+		 * TBD: uncomment function call below once node_free changes
+		 * done.
+		 */
+		/* btree_tx_node_capture(oi, bop->bo_tx); */
```

Comment:
Please rename this routine to btree_tx_capture() since there might be more than one nodes for this btree which would be captured.

Reply:
okay.